client/components/sites-dropdown/style.scss

```diff
@@ -65,6 +65,17 @@
 	}
 }
 
+.accessible-focus .sites-dropdown__selected {
+	svg.gridicons-chevron-down {
+		box-sizing: border-box;
+	}
```

Comment:
was this rule something we wanted all the time or only under accessible-focus?  The below rule involving :focus makes sense for using the accessible focus class, but did this sizing change only need to happen under a11y focus as well?

Reply:
Thanks for the catch! I hadn't tested with `box-sizing` just under the `focus` pseudoselector, but it works as expected. That's been updated 👍